### PR TITLE
Remove the voodoo doll

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Removes the voodoo doll, requested by @optimumtact.

## Why It's Good For The Game
A clunky to use grief tool that is near exclusively used to just make people say slurs and WGW and copypastas and rule breaking stuff and yadda yadda.

## Changelog
:cl:
del: Remove the voodoo doll.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/wizard/equipment/artefact.dm [hunk 1]
```diff
@@ -287,130 +287,6 @@
 	H.put_in_hands(new /obj/item/claymore(H), TRUE)
 	H.equip_to_slot_or_del(new /obj/item/spear(H), ITEM_SLOT_BACK)
 
-
-/obj/item/voodoo
-	name = "wicker doll"
-	desc = "Something creepy about it."
-	icon = 'icons/obj/wizard.dmi'
-	icon_state = "voodoo"
```
Comment (Timberpoes) [inline code/modules/antagonists/wizard/equipment/artefact.dm:295]: image

Doesn't look like anything else uses this icon_state. Worth doing a sweep over the icon files to clear it out as well? Any removed icons can always be pushed to the other icon repo or retrieved thanks to Github in general.

**Comment by Serxule** [comment]
"me remove meme item becuz it was used for meme" isn't really a good argument

**Review by Timberpoes — CHANGES_REQUESTED** [review]

**Comment by necromanceranne** [comment]
What about forced speech nanites?

**Comment by Mothblocks** [comment]
> "me remove meme item becuz it was used for meme" isn't really a good argument

that's okay

**Comment by Mothblocks** [comment]
> What about forced speech nanites?

I don't really like them either, but I'm keeping this within scope.

**Comment by ATH1909** [comment]
> "me remove meme item becuz it was used for meme" isn't really a good argument

to be fair, 'blocks dived through the logs of all the times that the voodoo doll has been used, and over 99% (not hyperbole) of the uses were just forcing people to shout slurs and rule 8 stuff.

I think that removing the entire doll is a bit overkill, though, so I'm gonna make a PR real quick.

File: code/modules/antagonists/wizard/equipment/artefact.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/antagonists/wizard/equipment/artefact.dm:295]: What you're referring to as github is actually git +gibhub, or as I've taken to calling it, gitgub

**Comment by Mothblocks** [comment]
For reference:

image

File: code/modules/antagonists/wizard/equipment/artefact.dm (hunk 1 above)
Comment (Timberpoes) [reply code/modules/antagonists/wizard/equipment/artefact.dm:295]: Gitgub also gets aliased to Gitgud too.

**Review by Timberpoes — APPROVED** [review]

**Comment by Qustinnus** [comment]
> "me remove meme item becuz it was used for meme" isn't really a good argument

good point, it shouldn't have been added in the first place, and you will be glad to know that crappy meme item PRs now get closed more often than not.

**Comment by EgorDinamit** [comment]
It should've not been a fucking lavaland loot, that's about it.
Ath has a better solution to the "They use it only for slurs" issue, but in any case - it shouldn't be a mining loot.

**Comment by Pugie** [comment]
Remove voice of god instead stupid retard coder *not insulting you (luv coders)*

**Comment by interestingusernam3** [comment]
Honestly this would fit into current wizard design well, but whoever added it to mining has a literal dent in his fucking head.

**Comment by ATH1909** [comment]
> It should've not been a fucking lavaland loot, that's about it.
> Ath has a better solution to the "They use it only for slurs" issue, but in any case - it shouldn't be a mining loot.

Eh, I think it's an alright fit for mining loot- it encourages miners to return to the station and interact with people. Not all mining loot needs to be about killing simplemobs.

**Comment by ModDownloading** [comment]
I don't think this warrants the voodoo doll being removed from the game code entirely, but I'm fine with making it a wizard item instead of leaving it in mining. The issue with making it look like people are saying strange stuff over the radio can be replicated by nanites, and to a lesser extent comms agents and lings that just change their voice to the target's. The other effects of the voodoo doll aren't problematic at all, so this shouldn't be flat-out removed.

**Comment by Mickyan** [comment]
I won't miss the forced speech but why remove the entire doll instead of just the problematic part? It's still a cool item

**Comment by davethwave** [comment]
Welp, orange just closed down the contesting PR's that would fix the voice thing. Sad, I liked the voodoo doll.

**Comment by ATH1909** [comment]
> Welp, orange just closed down the contesting PR's that would fix the voice thing. Sad, I liked the voodoo doll.

I'm not sure if he even read them, I think that he just saw that there was more than one competing PR and closed them immediately. He didn't even comment.

Which is a shame, because goof's PR was incredibly lazy and insane, and if he hadn't made it, mine might have had a chance.

**Pull request merged**